tests/lib/linter.js

```diff
@@ -2790,16 +2790,6 @@ describe("Linter", () => {
         });
     });
 
-    describe("when using invalid rule config", () => {
-        const code = TEST_CODE;
-
-        it("should throw an error", () => {
-            assert.throws(() => {
-                linter.verify(code, { rules: { foobar: null } });
-            }, /Invalid config for rule 'foobar'\./);
-        });
-    });
```

Comment:
I removed this test because we were doing config validation in two separate places: `ConfigValidator` which verifies all parts of a config including rule schemas, and `Linter` where we were only checking if a rule's configured value was `null` and skipping all other validation. The second validation is generally redundant with the first validation.